src/pages/trading-specification/components/_elements.tsx

```diff
@@ -135,23 +131,54 @@ export const TableHeaderCell = ({ text, infoIcon, toolTip }: TTableHeaderCell) =
     }
     return (
         <>
-            <Popover
-                isOpen={isInfoVisible}
-                positions={['top']}
-                padding={8}
-                content={
-                    <StyledToolTipContainer>
-                        <ToolTipText as="p" align="center">
-                            {toolTip}
-                        </ToolTipText>
-                    </StyledToolTipContainer>
-                }
-            >
-                <img src={infoIcon} onMouseOver={onMouseOver} onMouseLeave={onMouseLeave} />
-            </Popover>
+            {toolTip && (
+                <Popover
+                    isOpen={isInfoVisible}
+                    positions={['top', 'right', 'bottom', 'left']}
+                    padding={10}
+                    containerStyle={{ left: '-10px' }}
+                    content={({ position, childRect, popoverRect }) => (
+                        <ArrowContainer
+                            style={{
+                                backgroundColor: 'var(--color-grey-7)',
+                                borderRadius: '3px',
+                                padding: '5px',
+                            }}
+                            position={position}
+                            childRect={childRect}
+                            popoverRect={popoverRect}
+                            arrowColor={'var(--color-grey-7)'}
+                            arrowSize={10}
+                            arrowStyle={{ bottom: '-10px' }}
+                            className="popover-arrow-container"
+                            arrowClassName="popover-arrow"
+                        >
+                            <StyledToolTipContainer>
+                                <HeaderText
+                                    as="p"
+                                    align="center"
+                                    weight="normal"
+                                    type="extra-small"
+                                >
+                                    {toolTip}
+                                </HeaderText>
+                            </StyledToolTipContainer>
+                        </ArrowContainer>
+                    )}
+                >
+                    <img src={infoIcon} onMouseOver={onMouseOver} onMouseLeave={onMouseLeave} />
+                </Popover>
+            )}
 
             <Cell>
-                <StyledTableHeaderText type="small" width="fit-content" align="start" as="p">
+                <StyledTableHeaderText
+                    type="small"
+                    width="fit-content"
+                    align="start"
```

Comment:
if it possible, please provide this kind of properties inside `StyledTableHeaderText` 🙏